src/TypeScript/Analyzer/EslintBridgeIssueConverter.cs

```diff
@@ -44,8 +44,8 @@ public EslintBridgeIssueConverter(IRulesProvider rulesProvider)
         public IAnalysisIssue Convert(string filePath, Issue issue)
         {
             var ruleDefinitions = rulesProvider.GetDefinitions();
-            var ruleDefinition = ruleDefinitions.SingleOrDefault(x => x.EslintKey == issue.RuleId);
```

Comment:
The old null-handling behaviour was inconsistent - `SingleOrDefault` is used here, but then we'd have got null ref when accessing `ruleDefinition.Severity/Type` below.